server/submission/resolvers.js

```diff
@@ -189,16 +130,31 @@ const resolvers = {
       const newManuscriptDb = db.manuscriptGqlToDb(newManuscript, ctx.user)
       await db.update(newManuscriptDb, data.id)
 
-      const mailData = {
-        from: config.get('mailer.from'),
-        to: newManuscript.submissionMeta.author.email,
-        subject: 'Congratulations! You submitted your manuscript!',
-        text: 'Your manuscript has been submitted',
-        html: '<p>Your manuscript has been submitted</p>',
+      mailer
+        .send({
+          from: config.get('mailer.from'),
+          to: newManuscript.submissionMeta.author.email,
+          subject: 'Congratulations! You submitted your manuscript!',
+          text: 'Your manuscript has been submitted',
+          html: '<p>Your manuscript has been submitted</p>',
+        })
+        .catch(err => {
+          logger.error(`Error sending submitter confirmation email: ${err}`)
+        })
+
+      if (ctx.user.email !== manuscript.submissionMeta.author.email) {
+        mailer
+          .send({
+            from: config.get('mailer.from'),
+            to: newManuscript.submissionMeta.author.email,
+            text: 'Please verify that you are a corresponding author',
+          })
+          .catch(err => {
+            logger.error(
+              `Error sending corresponding author verification email: ${err}`,
+            )
+          })
```

Comment:
Do you think its worth pulling out a helper function for mailing?

Reply:
Probably not worth it. I refactored this code before I realised that we want to remove the first email anyway - see issue #240 

Since we only have one instance in which we want to send an email (at the moment), I figure we might as well not abstract yet?